pkg/colorful-french.man

```diff
@@ -0,0 +1,32 @@
+.\" Manpage for colorful
+.\" Contact veg@svgames.pl to correct errors or typos.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```

Comment:
"Games Manual" should also be translated.

Reply:
I checked with `man-pages(7)` and the date should reflect the time the manpage received substantial changes. A translation is rather substantial, so please use a current date. :)

Reply:
Fixed those remarks !